src/state/pair/types.ts

```diff
@@ -1,10 +0,0 @@
-export interface PopularPair {
```

Comment:
this isn't used anymore, is this?

Reply:
Yea, no reference found and no error when removing :D